ingestion_tools/scripts/common/mesh_converter.py

```diff
@@ -15,21 +17,91 @@ def convert_mesh_to_glb(convert_func: typing.Callable[[str], trimesh.Trimesh]) -
     :param convert_func: the function to convert a mesh file to a trimesh object.
     """
 
-    def wrapper(input_file: str, output_file: str, scale_factor: float = 1.0) -> None:
+    def wrapper(input_file: str, output_file: str, scale_factor: float = 1.0, *args, **kwargs) -> None:
         """
         Convert a mesh file to a glb file and scale it by a given factor.
 
         :param input_file: the path to a mesh file to convert.
         :param output_file: the name of the output file.
         :param scale_factor: the factor to scale the mesh by.
         """
-        mesh: trimesh.Trimesh = convert_func(input_file)
+        mesh: trimesh.Trimesh = convert_func(input_file, *args, **kwargs)
         mesh.apply_scale(scale_factor)
         mesh.export(output_file)
 
     return wrapper
 
 
+def check_mesh_name(input_file: str, name: str) -> str:
+    with h5py.File(input_file, "r") as fp:
```

Comment:
Could we add a comment that links to the schema documentation for h5?

Reply:
That information is in the from_hff function.